Dockerfiles/Dockerfile.webui

```
@@ -41,7 +42,7 @@ ENV TLDW_INTERNAL_API_ORIGIN=${TLDW_INTERNAL_API_ORIGIN}
 WORKDIR /app/apps/tldw-frontend
 
 RUN bun /app/apps/tldw-frontend/scripts/validate-networking-config.mjs
-RUN bun run build
+RUN bun run build:prod
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)